reference/catalog.v3.yml

```diff
@@ -21292,6 +21361,38 @@ components:
             calculated_weight:
               type: number
       x-internal: false
+      description: ''
+      x-examples:
+        example-1:
+          cost_price: 0
+          price: 0
+          sale_price: 0
+          retail_price: 0
+          weight: 0
+          width: 0
+          height: 0
+          depth: 0
+          is_free_shipping: true
+          fixed_cost_shipping_price: 0
+          purchasing_disabled: true
+          purchasing_disabled_message: string
+          upc: string
+          inventory_level: 0
+          inventory_warning_level: 0
+          bin_picking_number: string
+          mpn: string
+          gtin: '012345678905'
+          id: 0
+          product_id: 0
+          sku: string
+          sku_id: 0
+          option_values:
+            - option_display_name: Color
+              label: Beige
+              id: 146
+              option_id: 151
+          calculated_price: -1.7976931348623157e+308
```

Comment:
Was the calculated price intentional? We could make it 0

Reply:
I updated the value, so I'm resolving the convo